### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by DecCapri** [the1812/Bilibili-Evolved#4891] [提问 / Q&A] [answered]
虽然应该可能大概不是插件的问题，但还是想问一下，为什么我用chrome浏览器看B站直播中弹天选时刻的弹窗，背后会有一层阴影层，然后弹窗也点不了、关不掉，两台电脑都有这个问题，但换成edge又没事
![edf43ffe3ca7daecad39f48eae8a5e52](https://github.com/user-attachments/assets/63cc9096-49f5-469b-8f6c-ae6b1f86ca72)
![PixPin_2024-08-31_16-51-36](https://github.com/user-attachments/assets/56e88a7d-42be-4a42-90ad-0f51325a03e0)

**Comment by festoney8** [comment, marked as answer]
大概是网址参数清理导致的，试试关掉这功能，等天选出现再看

（天选用的 iframe 网址带参，被净化一下好像会出 bug）

**Reply by DecCapri** [reply]
没开这个，上次我把油猴直接关了这个问题都存在，所有很烦

**Reply by festoney8** [reply]
装过有网址净化功能的其他浏览器插件吗

**Reply by DecCapri** [reply]
没啊，我adguard都排除bilbili了，油猴也全关了

**Reply by DecCapri** [reply]
找到了，就是adguard的锅，但我明明排除了bilbil了，不知道为什么还会这样，谢谢了
